cl/alerts/tasks.py

```diff
@@ -122,3 +134,50 @@ def send_docket_alerts(data):
         send_docket_alert(*args)
 
     return data.get("rds_for_solr", [])
+
+
+@app.task(ind=True, max_retries=2, interval_start=5, interval_step=5)
+def crawl_pacer_mobile_page(self, d_pk):
+    cookies = get_or_cache_pacer_cookies(
+        "pacer_scraper",
+        username=settings.PACER_USERNAME,
+        password=settings.PACER_PASSWORD,
+    )
+    s = PacerSession(
+        cookies=cookies,
+        username=settings.PACER_USERNAME,
+        password=settings.PACER_PASSWORD,
+    )
+    d = Docket.objects.get(pk=d_pk)
+    report = MobileQuery(map_cl_to_pacer_id(d.court_id), s)
```

Comment:
Mobile site requires using the actual docket number, not the pacer id. As discussed, if we happen to not have the docket number we should kick off an iquery scrape to get it, which automatically updates it?